Utility/SQL/Migration.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# coding: utf-8
+
+import argparse
+from subprocess import Popen, PIPE
+import psycopg2
+import os.path
+
+PRODUCT_DATABASE_NAME = os.getenv('DATABASE_NAME')
+
+kill_db_processes_sql = """SELECT    pg_terminate_backend (pid)
+                        FROM    pg_stat_activity
+                        WHERE   datname = '{}';
+                    """.format(PRODUCT_DATABASE_NAME)
+
+new_database_name_sql = """ALTER DATABASE {0} RENAME TO _{0};
+                """.format(PRODUCT_DATABASE_NAME)
+
+
+def connection_parameters(db_name_env='POSTGRES_DB') -> str:
+    _args = "password=%s " % os.getenv('POSTGRES_PASSWORD') + \
```

Comment:
single underscore is used for internal variables/functions (i.e., they should not be called somewhere outside the class/file). The _args variable is local and the underscore at the beginning of the name is unnecessary.